layouts/partials/custom_head.html

```diff
@@ -5,6 +5,7 @@
   <meta name="go-import" content="gonum.org/v1/netlib git https://github.com/gonum/netlib">
   <meta name="go-import" content="gonum.org/v1/plot git https://github.com/gonum/plot">
   <meta name="go-import" content="gonum.org/v1/tools git https://github.com/gonum/tools">
+  <meta name="go-import" content="gonum.org/website git https://github.com/gonum/website/static/code">
```

Comment:
does that actually work?
I was under the impression it was expecting a repository root as a third argument.

https://golang.org/cmd/go/#hdr-Remote_import_paths

> The meta tag has the form:
> 
> ```
> <meta name="go-import" content="import-prefix vcs repo-root">
> ```

Reply:
Thanks, I was worried about that. I'll drop the second commit.

Reply:
It's a pity, it would be nice to have a clean import path for these things.

Reply:
yep. if we were to put some Go code in front (ie: a `net/http` router) we could control this (and fix the lingering issue of godoc redirection for _e.g.:_ `https://gonum.org/v1/gonum/mat#Matrix` (compared to _e.g._ `https://go-hep.org/x/hep/groot/rtree#Tree`) :))